drivers/flash/nrf_qspi_nor.c

```diff
@@ -506,60 +592,58 @@ static int qspi_nrfx_configure(const struct device *dev)
 	}
 #endif
 
+	/* Set QE to match transfer mode.  If not using quad
+	 * it's OK to leave QE set, but doing so prevents use
+	 * of WP#/RESET#/HOLD# which might be useful.
+	 *
+	 * Note build assert above ensures QER is S1B6 or
+	 * S2B1v1/4/5/6. Other options require more logic.
+	 */
 	if (INST_0_QER != JESD216_DW15_QER_NONE) {
-		/* Set QE to match transfer mode.  If not using quad
-		 * it's OK to leave QE set, but doing so prevents use
-		 * of WP#/RESET#/HOLD# which might be useful.
-		 *
-		 * Note build assert above ensures QER is S1B6.  Other
-		 * options require more logic.
-		 */
+		nrf_qspi_prot_conf_t const *prot_if =
+			&dev_config->nrfx_cfg.prot_if;
+		bool qe_value = (prot_if->writeoc == NRF_QSPI_WRITEOC_PP4IO) ||
+			(prot_if->writeoc == NRF_QSPI_WRITEOC_PP4O)  ||
+			(prot_if->readoc == NRF_QSPI_READOC_READ4IO) ||
+			(prot_if->readoc == NRF_QSPI_READOC_READ4O);
```

Comment:
I think it would be better to keep the indentation here as it was, i.e.:
```
		bool qe_value = (prot_if->writeoc == NRF_QSPI_WRITEOC_PP4IO) ||
				(prot_if->writeoc == NRF_QSPI_WRITEOC_PP4O)  ||
				(prot_if->readoc == NRF_QSPI_READOC_READ4IO) ||
				(prot_if->readoc == NRF_QSPI_READOC_READ4O);
```

Reply:
Fixed.